homeassistant/components/wilight/cover.py

```diff
@@ -18,6 +17,8 @@
 
 from . import WiLightDevice
 
+DOMAIN = "wilight"
```

Comment:
Thanks.  Would you please put it in `const.py` or `__init__.py` and import it where it needs to be used so we don't have duplicates.

Reply:
Done!